sql/parse_file.cc

```diff
@@ -354,6 +354,9 @@ sql_create_definition_file(const LEX_CSTRING *dir,
   @retval
     1   Error (only if renaming of frm failed)
 */
+/*
+commented out(357-387) for- https://jira.mariadb.org/browse/MDEV-23962
+
```

Comment:
When removing code, we actually remove it rather than comment it out. The code would be unreadable if there where just commented code. If we need the old copy of the code we look at the git history.

Reply:
Thank you @grooverdan for your review and pointing my mistakes.
Surely I'm now working on that points you've mentioned.